photutils/isophote/harmonics.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+import numpy as np
+from scipy.optimize import leastsq
```

Comment:
My initial understanding (was wrong) is that this is a requirement of the tests only. Now I notice that the code itself can't rely on scipy. But the scipy.optimize.leastsq function is at the very backbone of the entire isophote fitting idea. In a case like this, we cannot have scipy as an optional dependency; it's mandatory. Does it mean that the isophote code can't live in photutils after all?